rubicon-htb.js

```diff
@@ -254,6 +256,35 @@ function RubiconModule(configs) {
         return firstPartyData;
     }
 
+    function _getDigiTrustQueryParams() {
+        if (configs.digitrustId) {
+            _digiTrustId = configs.digitrustId;
+        }
+            function getDigiTrustId() {
+                if (!Browser.isTopFrame()) {
+                    try {
+                    var _window = window.top;
+                    } catch(e) {
+                        console.log("impossible to reach top window");
+                    }
+                } else {
+                    var _window = window;
```

Comment:
Please use `Broswer.topWindow` (Reference to the topmost accessible window context from where this library is)

Reply:
I don't have access to your Utility Libraries so not sure what stringToArray function is returning. I have coded one in libraryStubData. Could you please share the Utility Libraries? Thanks,